src/e2c/db_connector.py

```diff
@@ -11,31 +12,59 @@
 class DBConnector:
     """Database Connector."""
 
-    def __init__(self, path=_path):
+    def __init__(self, path: str = _path):
```

Comment:
We do not specify type hints. Instead, we write good docs to indicate this.

Reply:
Why not ? It will appear automatically in docs.